domain-management/src/main/java/org/jboss/as/domain/management/security/SecurityRealmService.java

```diff
@@ -97,6 +97,8 @@
  */
 public class SecurityRealmService implements Service<SecurityRealm>, SecurityRealm {
 
+    private static final Provider ELYTRON_PROVIDER = new WildFlyElytronProvider();
```

Comment:
Thinking about this I can probably get away with moving this into the service lifecycle, the reported problem was it was being created per request.